boards/FSM/Inc/Io/Io_WheelSpeedSensors.h

```diff
@@ -31,6 +31,26 @@ float Io_WheelSpeedSensors_GetRightSpeedKph(void);
  */
 void Io_WheelSpeedSensors_InputCaptureCallback(TIM_HandleTypeDef *htim);
 
+/**
+ * Gets the wheel speed in km/h from the left wheel speed sensor
+ * @return Returns the wheel speed in km/h
+ */
+float Io_WheelSpeedSensors_GetLeftSpeedKph(void);
+
+/**
+ * Gets the wheel speed in km/h from the right wheel speed sensor
+ * @return Returns the wheel speed in km/h
+ */
+float Io_WheelSpeedSensors_GetRightSpeedKph(void);
+
+/**
+ * Update the frequency for the left and right wheel speed sensors.
+ * @note This function should be called in the input capture callback function.
+ * @param htim The handle of the timer that triggered the input capture callback
+ * function
+ */
+void Io_WheelSpeedSensors_InputCaptureCallback(TIM_HandleTypeDef *htim);
+
```

Comment:
These are already added? Why are we writing them twice?

Reply:
This was likely introduced while rebasing my bad